src/Eclipse-IDE/org.robotframework.ide.eclipse.main.plugin/src/org/robotframework/ide/eclipse/main/plugin/tableeditor/source/SuiteSourceEditor.java

```diff
@@ -223,11 +225,13 @@ private void updateLineLocationStatusBar(final int caretPosition) {
     private void updateLineDelimitersStatus() {
         final IDocument document = getDocument();
 
-        final StatusLineContributionItem find = (StatusLineContributionItem) getEditorSite().getActionBars()
+        IContributionItem find = getEditorSite().getActionBars()
                 .getStatusLineManager()
-                .find(RobotFormEditorActionBarContributor.DELIMITERS_INFO_ID);
-        final String delimiter = DocumentUtilities.getDelimiter(document);
-        find.setText("\r\n".equals(delimiter) ? "CR+LF" : "LF");
+                .find(ITextEditorActionConstants.STATUS_CATEGORY_INPUT_POSITION);
+        if (find instanceof StatusLineContributionItem) {
+            final String delimiter = DocumentUtilities.getDelimiter(document);
+            find.setText("\r\n".equals(delimiter) ? "CR+LF" : "LF");
```

Comment:
Does this compile?

Reply:
Surely it will not compile, similarly as line 218

Reply:
@michalanglart 
Sorry, Can you update as below for me ? Thanks.
`private void updateLineLocationStatusBar(final int caretPosition) {
        try {
            final IDocument document = getDocument();
            int lineNumber = document.getLineOfOffset(caretPosition);
            final int columnNumber = caretPosition - document.getLineOffset(lineNumber) + 1;
            lineNumber++;

            final IContributionItem find = getEditorSite().getActionBars()
                    .getStatusLineManager()
                    .find(ITextEditorActionConstants.STATUS_CATEGORY_INPUT_POSITION);
            if (find instanceof StatusLineContributionItem) {
				((StatusLineContributionItem )find).setText(lineNumber + ":" + columnNumber);
			}
        } catch (final BadLocationException e) {
            RedPlugin.logError("Unable to get position in source editor in order to update status bar", e);
        }
    }

    private void updateLineDelimitersStatus() {
        final IDocument document = getDocument();

        final IContributionItem find = getEditorSite().getActionBars()
                .getStatusLineManager()
                .find(RobotFormEditorActionBarContributor.DELIMITERS_INFO_ID);
		if (find instanceof StatusLineContributionItem) {
			final String delimiter = DocumentUtilities.getDelimiter(document);
			((StatusLineContributionItem )find).setText("\r\n".equals(delimiter) ? "CR+LF" : "LF");
		}
    }`